SPECS/util-linux.spec

```
@@ -2,7 +2,7 @@
 Summary: A collection of basic system utilities
 Name: util-linux
 Version: 2.23.2
-Release: 52%{?dist}.1
+Release: 52.0yd1%{?dist}
```

As defined in https://github.com/xcp-ng/xcp/wiki/RPM-Packaging-guidelines (I know, still has to be moved to official docs), the temporary release number should be `52.0.yd.1%{?dist}`, or something like `52.0.sectorsize.1%{?dist}`

```suggestion
Release: 52.0.yd.1%{?dist}
```

---

ah, dots... done